src/extension/android/androidPlatform.ts

```diff
@@ -235,16 +252,11 @@ export class AndroidPlatform extends GeneralMobilePlatform {
             }
             if (this.runOptions.target) {
                 if (
-                    this.runOptions.target === AndroidPlatform.simulatorString ||
-                    this.runOptions.target === AndroidPlatform.deviceString
+                    !(
+                        this.runOptions.target === TargetType.Device ||
+                        this.runOptions.target === TargetType.Simulator
+                    )
```

Comment:
I think this expression can be simplified a little
```suggestion
                    this.runOptions.target !== TargetType.Device &&
                    this.runOptions.target !== TargetType.Simulator
```